### ISSUE TITLE
I changed my config but it didn't work

### ISSUE THREAD

**Issue by ninjaguardian** [Corosauce/WATUT#44]
**Describe the bug**
I was looking at https://github.com/Corosauce/WATUT/issues/8 and I followed the instructions but it did not work.

(also by the way there is a typo in this template arent is spelled aren't)

**To Reproduce**
1. Go to watut-client.toml
2.
```toml
showIdleStatesInPlayerList = false
```
3. Launch game with better ping display
4. Crash

**latest.log, debug.log or crash report if the other 2 arent possible**
https://mclo.gs/4yihQ7C

**Expected behavior**
It should disable the feature and not crash

**Mod and Minecraft version:**
 - Version: 1.20.1-1.1.3

**Comment by ninjaguardian** [comment]
When showIdleStatesInPlayerList is false, renderPingIconHook returns false. When renderPingIconHook is false, renderPingIcon renders the icon. So it is wrong.

src/main/java/com/corosus/watut/mixin/client/RenderPingIconInject.java line 23 does 

```Java
if (!WatutMod.getPlayerStatusManagerClient().renderPingIconHook((PlayerTabOverlay)(Object)this, pGuiGraphics, p_281809_, p_282801_, pY, pPlayerInfo)) {…}
```

but it should do

```Java
if (WatutMod.getPlayerStatusManagerClient().renderPingIconHook((PlayerTabOverlay)(Object)this, pGuiGraphics, p_281809_, p_282801_, pY, pPlayerInfo)) {…}
```

But, of course, I could be wrong.

**Comment by Corosauce** [comment]
Yeah I dunno why that suggestion worked for other people, the mixin will still run regardless of a config, either way this issue is fixed now with this push: 

Fixed with https://github.com/Corosauce/WATUT/commit/2d3e5f10350d7ee98ca727a27c724724b6aaa59e - will be fixed in watut 1.2.0+, will make sure better ping display works and my change in the code gracefully deactivates, will get ported to 1.21 version at a later time.

As for the second comment I checked just in case my dyslexia got the better of me that day, but line 23 is correct as is, it returns false when my code has no reason to run, which then lets vanilla do its thing.
